jwk/handler_test.go

```diff
@@ -53,6 +53,9 @@ func TestHandlerWellKnown(t *testing.T) {
 	JWKPath := "/.well-known/jwks.json"
 
 	t.Run("Test_Handler_WellKnown/Run_public_key_With_public_prefix", func(t *testing.T) {
```

Comment:
Is it possible to create a similar test with HSM enabled? Or is `Run_public_key_Without_public_prefix` testing this sufficiently?